docs/docs/index.md

```diff
@@ -8,6 +8,7 @@ license: CC-BY-SA-4.0
 Welcome to Fragcolor documentation. Here, you can find the documentation of the technologies used with Fragcolor.
 
 - [Chainblocks](./blocks/)
+- [Contribution guide](./contribute/)
```

Comment:
Maybe "contributions guides" since there are several, although one could argue the whole section is a big guide.

Reply:
Actually, yes, I agree. There are at least three separate guides in there (code, docs, wiki) so "contributions guides" is a better heading.
Fixed.